### PR TITLE
new dlgImportGriddedData

### PR DESCRIPTION
**Pull request by shadrackkibet**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons this now needs to be merged as its a new dialog and contains form main and vb proj. 

Thanks

File: instat/frmMain.Designer.vb [hunk 1]
```diff
@@ -1421,44 +1422,44 @@ Partial Class frmMain
...
         'ToolStripSeparator15
         '
         Me.ToolStripSeparator15.Name = "ToolStripSeparator15"
-        Me.ToolStripSeparator15.Size = New System.Drawing.Size(153, 6)
+        Me.ToolStripSeparator15.Size = New System.Drawing.Size(203, 6)
         '
         'mnuClimaticFileExportToCPT
         '
         Me.mnuClimaticFileExportToCPT.Name = "mnuClimaticFileExportToCPT"
-        Me.mnuClimaticFileExportToCPT.Size = New System.Drawing.Size(156, 22)
```
Comment (dannyparsons) [inline instat/frmMain.Designer.vb]: This doesn't look right, these things shouldn't be changing size. @maxwellfundi can you check this branch on your machine to see if there are changes to these items?

Comment (dannyparsons) [inline instat/frmMain.Designer.vb]: Or unless this is because the new item has a longer name. This is probably fine.

Comment (maxwellfundi) [inline instat/frmMain.Designer.vb]: okay, let me check that now..

Comment (maxwellfundi) [inline instat/frmMain.Designer.vb]: Hey @dannyparsons , the menu is fine. I the new one is the one that is longer than the others so they had to be of same length. 

thanks.

**Pull request merged**